core/templates/pages/admin-page/services/admin-jobs-tab-backend-api.service.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Backend Api Service for jobs tab in the admin panel.
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { AdminPageConstants } from '../admin-page.constants';
+import { AdminDataService } from './admin-data.service';
+
+export interface AdminJobOutputResponse {
+  output: string[]
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdminJobsTabBackendApiService {
```

Comment:
All admin tabs should have a single backend-api.service (` admin-backend-api.service.ts`)
@DubeySandeep suggested no need for multiple backend-api files.
@DubeySandeep any thoughts?

Reply:
Someone created a *backend-api.service.ts for admin-config-tab earlier, So, I repeated the same pattern.

Reply:
https://github.com/oppia/oppia/pull/11463
It is also being changed.

Reply:
Thanks, @Radesh-kumar, but need a nod on this from maintainers.

Reply:
I am fine with this. @ashutoshc8101 can you coordinate with @Radesh-kumar on merging the services. I think this will be just copying the functions into one file but if you need help with anything just let me know.

Reply:
Admin jobs tab has a service in here https://github.com/oppia/oppia/blob/8013e429dcbb7e40d72406c077812dff38f211cd/core/templates/domain/admin/admin-backend-api.service.ts#L238
No need for adding the same thing here [It will be redundant].